app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,81 @@
  *= require_tree .
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
Thank you.